zerver/tests/test_auth_backends.py

```diff
@@ -4679,11 +4679,9 @@ def test_update_user_avatar_for_s3(self) -> None:
         self.change_ldap_user_attr('hamlet', 'jpegPhoto', b'00' + test_image_data)
         with self.settings(AUTH_LDAP_USER_ATTR_MAP={'full_name': 'cn',
                                                     'avatar': 'jpegPhoto'}):
-            with mock.patch('logging.warning') as mock_warning:
+            with self.assertLogs(level="WARNING") as m:
                 self.perform_ldap_sync(self.example_user('hamlet'))
-                mock_warning.assert_called_once_with(
-                    'Could not parse %s field for user %s', 'jpegPhoto', hamlet.id,
-                )
+            self.assertEqual(m.output, [f"WARNING:root:Could not parse jpegPhoto field for user {hamlet.id}"])
```

Comment:
same here with `jpegPhoto`. We want to avoid having it hidden as a part of the log because when someone wants to find the code generating that log they might search for `Could  not parse jpegPhoto field...` and will not be able to find it because `jpedPhoto` won't be there in the actual code.